templates/overrides/semantic/webbasic/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,103 @@
+﻿@using System.Security.Claims
+<!doctype html>
+<html lang="">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <title>@ViewData["Title"] - SemanticUI</title>
+
+    <environment names="Development">
+        <link rel="stylesheet" href="~/lib/semantic/dist/semantic.css" />
+        <link rel="stylesheet" href="~/css/site.css" />
+    </environment>
+    <environment names="Staging,Production">
+        <link rel="stylesheet" href="//oss.maxcdn.com/semantic-ui/2.1.8/semantic.min.css"
+              asp-fallback-ref="~/lib/semantic/dist/semantic.min.css"
+              asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
+
+        <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
+    </environment>
+</head>
+<body>
+
+
+    <!-- Following Menu -->
+    <div class="ui large top pointing fixed menu">
+        <div class="ui container">
```

Comment:
This is not exact port of Templates navigation patter. There is no `header` element in menu - which is something similar to Bootstrap bran element available in aspnet\Templates
This should use `menu.header`"
https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/collections/menu.less#L182-L198
As in Semantic example:
http://semantic-ui.com/examples/fixed.html (navbar)